### PR TITLE
Issue/1923: Options to disable each email

### PR DESCRIPTION
**Pull request by ramiabraham**
- Adds options to disable additional affiliates emails via `disable_application_accepted_emails`, `disable_new_referral_emails`.

- Moves `registration_notifications` setting to be underneath other disable email options.

- Adds checks for new disabled emails settings in email functions `affwp_notify_on_new_referral` and `affwp_notify_on_approval`.

- #1923

### CODE REVIEW

File: includes/emails/actions.php [hunk 1]
```diff
@@ -82,6 +82,10 @@ function affwp_notify_on_registration( $affiliate_id = 0, $status = '', $args =
  */
 function affwp_notify_on_approval( $affiliate_id = 0, $status = '', $old_status = '' ) {
 
+	if( ! affiliate_wp()->settings->get( 'disable_application_accepted_emails' ) ) {
```
Comment (pippinsplugins) [inline includes/emails/actions.php]: This logic is backwards. It should be `if( affiliate_wp()->settings->get( 'disable_application_accepted_emails' ) ) {` here and all cases below.

**Comment by ramiabraham** [comment]
@pippinsplugins Corrected in https://github.com/AffiliateWP/AffiliateWP/pull/2373/commits/fc10dd24e711ca1adab7480d56ae7b18f6c42fba

File: includes/emails/actions.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
  */
 function affwp_notify_on_registration( $affiliate_id = 0, $status = '', $args = array() ) {
 
-	if( ! affiliate_wp()->settings->get( 'registration_notifications' ) ) {
+	if( affiliate_wp()->settings->get( 'registration_notifications' ) ) {
```
Comment (pippinsplugins) [inline includes/emails/actions.php]: Sorry this one was correct. The email is sent if the option IS checked.

Comment (ramiabraham) [reply includes/emails/actions.php]: @pippinsplugins Corrected!

**Pull request merged**